### ISSUE TITLE
Submission on Argoverse 1 Leaderboard

### ISSUE THREAD

**Issue by hanm53326** [fudan-zvg/DeMo#26]
Firstly, thanks for the contribution of your work. 

I would like to ask have you ever submitted your evaluation results on Argoverse 1 Leaderboard?

I followed your data extractor and data-module setting, and trained the DeMo on AV1. I can get similar evaluation results on AV1 VAL set as listed in your paper. However, when i submitted the evaluation results on test set(to be precisely, test_obs set in AV1), the results is bad. It seems overfitting happens.

Can you help look into this issue? thanks.
